### PR TITLE
Issue101 backend support for rating

### PR DESCRIPTION
**Pull request by ahmedqarmout2**
Issue101 backend support for rating

### CODE REVIEW

File: server/users.js [hunk 1]
```diff
@@ -184,6 +186,12 @@ exports.getQuestionsList = function(request, callback) {
 	db.getQuestionsListByUser(request, callback);
 }
 
+// set the atatus of the user to active or in-active
```
Comment (esteeroramy) [inline server/users.js]: typo `atatus` -> `status`

**Review by esteeroramy — APPROVED** [review]

**Comment by esteeroramy** [comment]
looks good 👍

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Nice work

**Review by Mohammed-JS-Khan — APPROVED** [review]
I rate it :100: / :100:

**Review by mehtadarshan1 — APPROVED** [review]

**Pull request merged**